lib/socialWidgetHandler.js

```diff
@@ -60,7 +60,7 @@ exports.getSocialWidgetContentScriptData = function(tab) {
   socialWidgetList.forEach(function(socialWidget) {
     let socialWidgetName = socialWidget.name;
 
-    if (storage["userGreen"].hasOwnProperty(socialWidget.domain)){
+    if (storage.userGreen.hasOwnProperty(socialWidget.domain)){
```

Comment:
I changed it because jshint was complaining about it, functionally it does the exact same thing.